packages/metrics-handler/index.js

```diff
@@ -0,0 +1,57 @@
+function responseHelper(body) {
+  return {
+    statusCode: 200,
+    body: JSON.stringify(body),
+  };
+}
```

Comment:
this name is not communicating the intention very well. Maybe let's call it `createOkResponse` or something similar?

Reply:
😂 I wrote this and it's something I use in my other projects. It usually accepts a status and body but for this use case, we only want to send 200s. I think the rename is fine but just sharing the history.

Reply:
renamed to `createOkResponse`